src/app/portfolio/AssetsBreakdown.tsx

```diff
@@ -166,7 +166,7 @@ export const AssetsBreakdown: React.FC<{
                     htmlFor="hideBalanceAssetsBreakdown"
                     className="text-sm font-medium leading-none peer-disabled:cursor-not-allowed peer-disabled:opacity-70"
                   >
-                    {`Hide low balances (< 1$)`}
+                    {`Hide chains with low balances (< 1$)`}
```

Comment:
chains or assets ? to keep naming consistent

Reply:
(Also maybe a bit long, I find the current label more consise)

Reply:
Will merge as is and modify the wording once it's moved to the setting page 😉